src/chat.rs

```diff
@@ -1937,8 +1937,9 @@ impl Chat {
     pub(crate) async fn sync_contacts(&self, context: &Context) -> Result<()> {
         let mut addrs = Vec::new();
         for contact_id in get_chat_contacts(context, self.id).await? {
-            let contact = Contact::get_by_id(context, contact_id).await?;
-            addrs.push(contact.get_addr().to_string());
+            if let Some(contact) = Contact::get_by_id_optional(context, contact_id).await? {
```

Comment:
Done. Otherwise if a contact is added to the chat and another one is removed from the db then, it's possible that the addition wouldn't be synced but the removal would be. Although such a race is very unlikely, users are not so fast unless they are bots